backend/Dockerfile

```diff
@@ -5,12 +5,15 @@ FROM node:20.2-alpine as builder
 WORKDIR /app
 
 COPY package.json package-lock.json tsconfig.json ./
+COPY prisma ./prisma
 RUN npm install
+# RUN npx prisma generate
```

Comment:
Why is prisma generate not being run in the build stage?

Reply:
I've updated it to run in both stages now, I thought it might not be necessary during the build stage so I had commented it out. Is this fine now? 🤔

Reply:
Is it required to actually build the app? I just assumed it was, hence I asked about it (as prisma injects typescript information from my understanding).

Reply:
![image](https://github.com/csesoc/cselectives-v2/assets/48422312/07884886-12fc-423f-b0d2-ea041a9d97cb)
Oh nvm it's because prisma isn't being used properly gg; please address this before fully converting this project to prisma ‼️

Reply:
Nah i dont think the container uses the prisma generate to build first?? (it works fine without it)

yup the code gets quite problematic without the rawquery for courses. Pure prisma should be super doable for all other services though. 
Unless you can see a cool way around it? 👀

Reply:
I don't extensively use prisma so I'm not sure if there's a pure prisma way to do what you're trying to achieve, unfortunately - do you see pure prisma being used later down the line for other things?

Reply:
yup, all other services will be pure prisma, i tried to make course pure prisma too but n^26 time complexity wasnt worth it 💀